crates/sui-types/src/error.rs

```diff
@@ -428,13 +427,7 @@ pub enum SuiError {
     SuiSystemStateUnexpectedVersion,
 
     #[error("Message version is not supported at the current protocol version")]
```

Comment:
nit: Can we expose the `error` message in the `thiserror` output?

```suggestion
    #[error("Message version is not supported at the current protocol version: {error}")]
```